src/components/Navbar.tsx

```diff
@@ -1,4 +1,12 @@
+import { NavLink } from 'react-router-dom';
+import cn from 'classnames';
+
 export const Navbar = () => {
+  const getLinkClass = ({ isActive }: { isActive:boolean }) => cn('navbar-item',
+    {
+      'has-background-grey-lighter': isActive,
+    });
```

Comment:
move it outside the component (you can put before Navigation component or to utils folder) because this is hardcoded code which will never change so there is no need to create it each time per each render in the component